libraries/botbuilder-azure/src/azureBlobTranscriptStore.ts

```diff
@@ -323,7 +364,11 @@ export class AzureBlobTranscriptStore implements TranscriptStore {
         return AzureBlobTranscriptStore[checkedCollectionsKey][key];
     }
 
-    private createBlobService({storageAccountOrConnectionString, storageAccessKey, host}: BlobStorageSettings): BlobServiceAsync {
+    /**
+     * Create a Blob Service.
+     * @param param0 Settings required for configuring the Blob Service (storage account or connection string, access key and host).
```

Comment:
Is this param name correct?

Reply:
I found this, https://jsdoc.app/tags-param.html#parameters-with-properties for destructuring objects in parameters 'Documenting a destructuring parameter'

Reply:
Feedback applied. Thanks!